test/topology_tasks/test_tablet_tasks.py

```diff
@@ -12,7 +12,7 @@
 from test.pylib.repair import create_table_insert_data_for_repair, get_tablet_task_id
 from test.pylib.tablets import get_all_tablet_replicas
 from test.topology.conftest import skip_mode
-from test.topology_experimental_raft.test_tablets import inject_error_on
+from test.topology_custom.test_tablets2 import inject_error_on
```

Comment:
I'm not a big fan of have two test files, test_tablets and test_tablets2. But we can fix this later.

Reply:
Yeah. I don't know why have two in the first place. But I don't own these files.